edgelet/edgelet-http/src/pid.rs

```diff
@@ -88,3 +97,31 @@ mod impl_unix {
         }
     }
 }
+
+#[cfg(windows)]
+use self::impl_windows::get_pid;
+
+#[cfg(windows)]
+mod impl_windows {
+    use std::io;
+    use std::os::windows::io::AsRawSocket;
+    use winapi::ctypes::c_long;
+    use winapi::um::winsock2::{ioctlsocket, SOCKET_ERROR, WSAGetLastError};
+
+    use super::*;
+
+    const SIO_AF_UNIX_GETPEERPID: c_long = 0x5800_0100;
```

Comment:
Please add a comment which header this came from, for future reference.

Reply:
Good idea. It's from the new afunix.h in the Windows SDK. I'll add a comment to that effect.